website/static/js/registrationUtils.js

```diff
@@ -516,6 +534,15 @@ var RegistrationEditor = function(urls, editorId) {
         }
     });
 
+    self.markAsRequired = function() {
```

Comment:
All instance methods should go on the prototype, i.e. Comment.prototype.markAsRequired